### DISCUSSION TITLE
Triggering workflows from a Service task

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#223] [Q&A] [answered]
Hi @ralphhanna,

Say I have a Service Task in `appServices.ts` which would trigger new workflows automatically based on the current workflow's data.

How to achieve this? Thanks.

**Comment by ralphhanna** [comment, marked as answer]
Hi @MaheshkumarSundaram 

the service task would something like this:

```ts
    async DummyService1(input, context) {
        context.item.data.service1Result = 'Service1Exec';
        let gdt=context.execution.instance.data;
        let idt=context.item.data;
        console.log('global data:',gdt,'item data',idt);

        context.appDelegate.server.engine.start("Buy Used Car",{caseId:gdt.caseId})
        
    }
```

**Reply by MaheshkumarSundaram** [reply]
@ralphhanna,

Thanks. That works. 

But what's difference between `context.execution.instance.data` and `context.item.data`?
Because the console.log is the same for both?

**Comment by ralphhanna** [comment]
In some cases it is the same, but for subprocesses or loops it will be different.
`context.execution.instance.data` is the for the root
